searches/query.js

```diff
@@ -0,0 +1,77 @@
+const _ = require("lodash");
+const sample = require("../samples/sample_query");
+
+const searchQuery = async (z, bundle) => {
+
+  // List all the workspaces the actor has access to
+  const listWorkspacesResponse = await z.request({
+    method: "GET",
+    url: `https://cloud.steampipe.io/api/latest/actor/workspace`,
```

Comment:
Will this always be cloud.steampipe.io? Or could the host be different, since we take in the `cloud_host` as well which allows users to pass in another URL.